packages/react/src/primitives/types/datastore.ts

```diff
@@ -44,6 +44,6 @@ export type DataStorePredicateObject = {
   and?: DataStorePredicateObject[];
   or?: DataStorePredicateObject[];
   field?: string;
-  operand?: string;
+  operand?: string | boolean | number;
```

Comment:
Is there a type from `DataStore` that we could use here?

Reply:
as far as I can tell, datastore is using `any` for operand type. So I'd say no.

Reply:
I mean, `any` is a type right? 😅

Reply:
Yeah, looks like this is the PredicateObject type:

```
export type PredicateObject<T extends PersistentModel> = {
	field: keyof T;
	operator: keyof AllOperators;
	operand: any;
};
```
https://github.com/aws-amplify/amplify-js/blob/main/packages/datastore/src/types.ts#L780

We've tested that `boolean` and `number` work correctly I presume?